darknet.py

```diff
@@ -61,26 +41,144 @@ class DETECTION(Structure):
                 ("objectness", c_float),
                 ("sort_class", c_int),
                 ("uc", POINTER(c_float)),
-                ("points", c_int)]
+                ("points", c_int),
+                ("embeddings", POINTER(c_float)),
+                ("embedding_size", c_int),
+                ("sim", c_float),
+                ("track_id", c_int)]
```

Comment:
> PS: you have done a lot of changes since last time (congratulations!). If closing this pull request and opening a new one is better for you, let me know. I have developed this code with your last commit.

You have added all the necessary changes, so this should work successfully.